### PR TITLE
Update to `1.0.3`

### PR DESCRIPTION
**Pull request by dmitrykuzmin**
This PR updates the Spine version to `1.0.3`.

It also brings Gradle Wrapper update to `5.6.1` as well as some library version updates.

### CODE REVIEW

**Comment by dmitrykuzmin** [comment]
@dmdashenkov PTAL.

File: src/main/java/io/spine/helloworld/SimpleStorageFactory.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * A simple in-memory storage factory.
+ *
+ * <p>While such storage factory is sufficient for a "Hello World"-type application, it's not
+ * eligible for usage in a production environment.
+ *
+ * <p>For the production scenarios consider using
+ * <a href="https://github.com/SpineEventEngine/gcloud-java/">Spine library for
+ * Google Cloud Datastore</a>.
+ */
+final class SimpleStorageFactory implements StorageFactory {
```
Comment (dmdashenkov) [inline src/main/java/io/spine/helloworld/SimpleStorageFactory.java]: I don't like the idea of creating a storage implementation in the simplest example for Spine.

I think, we'd be better off switching the environment to "test mode". That we at least could explain to the first-time beholders.

File: src/main/java/io/spine/helloworld/Application.java [hunk 2]
```diff
@@ -53,10 +55,20 @@
...
         this.requestFactory = createRequestFactory();
     }
 
+    /**
+     * Configures conditions and configuration under which the application operates.
+     */
+    private static void configureServerEnvironment() {
+        ServerEnvironment serverEnvironment = ServerEnvironment.instance();
+        serverEnvironment.configureStorage(new SimpleStorageFactory());
+        serverEnvironment.configureTransport(InMemoryTransportFactory.newInstance());
```
Comment (dmdashenkov) [inline src/main/java/io/spine/helloworld/Application.java]: Isn't this the default?

File: gradle/static-analysis.gradle [hunk 3]
```diff
@@ -29,7 +29,7 @@ apply plugin: 'pmd'
  * See https://github.com/gradle/gradle/issues/8277.
  */
 pmd {
-    toolVersion = "${deps.versions.pmd}"
+    toolVersion = '6.16.0'
```
Comment (dmdashenkov) [inline gradle/static-analysis.gradle]: Why don't we use that version any more?

File: .idea/gradle.xml [hunk 4]
```diff
@@ -4,12 +4,12 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="createEmptyContentRootDirectories" value="true" />
         <option name="delegatedBuild" value="false" />
         <option name="testRunner" value="PLATFORM" />
         <option name="distributionType" value="WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="1.8" />
+        <option name="gradleHome" value="D:/Download/gradle-5.6.1" />
```
Comment (dmdashenkov) [inline .idea/gradle.xml]: I don't think this file should be here at all.

File: gradle/static-analysis.gradle (hunk 3 above)
Comment (dmitrykuzmin) [reply gradle/static-analysis.gradle]: The `deps` property is no longer exposed by the `bootstrap` plugin, starting from around version `0.15.7`. 

There seems to be nothing in its place to acquire the dependency versions from.

**Comment by dmitrykuzmin** [comment]
@dmdashenkov PTAL again.

**Pull request merged**